src/feedbacks.ts

```diff
@@ -0,0 +1,75 @@
+import { CompanionFeedbacks, CompanionFeedbackEvent, CompanionFeedbackResult } from "../../../instance_skel_types"
+import InstanceSkel = require("../../../instance_skel")
+import { MagewellConfig } from "./config"
+import { DeviceStatus } from "./magewell"
+import { MagewellState } from "./magewellstate"
+
+export enum FeedbackId {
+  Stream = 'stream',
+  Record = 'record'
+}
+
+export function GetFeedbacks(instance: InstanceSkel<MagewellConfig>, state: MagewellState): CompanionFeedbacks {
+  return {
+    [FeedbackId.Record]: {
+      label: 'Change color based on Record status',
+      description: 'If the device is recording change the color',
+      options: [
+        {
+          type: 'colorpicker',
+          label: 'Color Live',
+          id: 'color_live',
```

Comment:
feedbacks normally have a foreground colour input too, so that any text can still be readable